docs/user/openstack/provider_networks.md

```diff
@@ -0,0 +1,123 @@
+# Provider Networks
+
+
+## Table of Contents
+- [Provider Networks](#provider-networks)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Common prerequisites](#common-prerequisites)
+  - [Considerations when creating provider networks](#considerations-when-creating-provider-networks)
+  - [Deploying cluster with primary interface on a provider network with IPI](#deploying-cluster-with-primary-interface-on-a-provider-network-with-ipi)
+  - [Known issues](#known-issues)
+
+
+## Introduction
+
+Provider networks map directly to an existing physical network in the data center.
+Useful network types in this category include flat (untagged), VLAN (802.1Q tagged) and VXLAN. 
+OpenShift clusters can be deployed on provider networks, so the installer won't need to create
+tenant networks nor floating IPs.
+More information can be found about provider networks terminology [here][1].
+
+Here is a basic architecture of one OCP cluster running on a provider network and another one
+on a tenant network:
+
+![OCP on a provider network](provider-network.png)
+
+
+## Common prerequisites
```

Comment:
Not sure if "Common" is used commonly enough in this or related repos to work here. What are you trying to convey? 

Are there different configuration paths? 

If not:

```suggestion
## Prerequisites
```

Reply:
:+1: